scripts/008_scored_survey.R

```diff
@@ -10,13 +10,15 @@ library(tidyverse)
 library(ggradar)
```

Comment:
add a simple example of what each library is doing. `library(scales)` is not being used anymore

Reply:
on it

Reply:
fixed